### PR TITLE
Translating some of the last LP stories

### PR DESCRIPTION
**Pull request by hannahbergam**
UnassignSectionButton - Super simple. I think could be helpful in contrast to the multiple assign button, but could be convinced to delete it instead. 
Screenshot 2023-05-25 at 9 54 58 AM

LessonStatusList was hard to translate and verify. I checked some of the standards views in the progress tab and this might actually be correct.
Example:
Screenshot 2023-05-25 at 11 09 27 AM

Finally, this PR translates the Teacher Homepage. This one is tricky, as the view has changed quite a bit (and has been refactored) since the last time this storybook worked. These seem to work as expected, but there seems to be something amiss where they're all being rendered as if the sections are PL sections. I'd love any input on what might not be working here.

Before:
Screenshot 2023-05-25 at 3 03 52 PM

After:
Screenshot 2023-05-25 at 3 00 38 PM

Screenshot 2023-05-25 at 3 00 27 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/TEACH-516

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/UnassignSectionButton.story.jsx [hunk 1]
```diff
@@ -4,16 +4,15 @@ import {UnconnectedUnassignSectionButton as UnassignSectionButton} from './Unass
...
-      ),
-    },
-  ]);
+
+export default {
+  name: 'UnassignSectionButton',
+  component: UnassignSectionButton,
 };
+
+export const WithSubtitle = () => (
```
Comment (bethanyaconnor) [inline apps/src/templates/UnassignSectionButton.story.jsx]: Where did this name come from?

File: apps/src/templates/studioHomepages/TeacherHomepage.story.jsx [hunk 2]
```diff
@@ -224,4 +163,9 @@ function withFakeServer({courses = [], sections = []} = {}) {
     '/dashboardapi/sections/valid_course_offerings',
     successResponse([])
   );
+  server.respondWith(
```
Comment (bethanyaconnor) [inline apps/src/templates/studioHomepages/TeacherHomepage.story.jsx:167]: Wow, these have been broken for a really long time haven't they

**Review by bethanyaconnor — APPROVED** [review]
one minor question, otherwise looks good!

File: apps/src/templates/studioHomepages/TeacherHomepage.story.jsx (hunk 2 above)
Comment (hannahbergam) [reply apps/src/templates/studioHomepages/TeacherHomepage.story.jsx:167]: I believe so😅

File: apps/src/templates/UnassignSectionButton.story.jsx (hunk 1 above)
Comment (hannahbergam) [reply apps/src/templates/UnassignSectionButton.story.jsx]: LOL good question. Had to be from a copy/paste. Updating now!

**Pull request merged**